XiEditor/Document.swift

```diff
@@ -94,7 +93,9 @@ class Document: NSDocument {
         windowController.window?.setFrame(newFrame, display: true)
         windowController.window?.minSize = Document.minWinSize
 
-        self.editViewController = windowController.contentViewController as? EditViewController
+        guard let splitViewController = windowController.contentViewController as? NSSplitViewController else { return }
```

Comment:
It seems like there's a bug here where at least 2 windows have to be opened before the initial window is shown. On current master, when opening xi-mac I get an "Untitled" window, but with this branch there is sometimes a dummy "Untitled 2" window before I can interact with a "Untitled 3" window.

Reply:
I believe this should be fixed now